src/components/sky-nav/sky-nav.scss

```diff
@@ -357,6 +378,14 @@ $_masthead-height-sm: ms.step(7);
     display: flex; /* 1 */
     flex: 1; /* 2 */
     justify-content: flex-end; /* 3 */
+
+    /**
+     * For larger viewports, we do the opposite of smaller viewports and show
+     * the menu items during the "loading" state.
+     */
+    .is-loading & {
+      display: flex;
+    }
```

Comment:
Instead of applying this by default and then overriding at a larger breakpoint, could you use a `@media (width < $_breakpoint-wide)` media query on the original `display: none` style?

Reply:
Good idea! Let me give it a try. 🏗️

Reply:
Thanks for the suggestion, @tylersticka! https://github.com/cloudfour/cloudfour.com-patterns/pull/1947/commits/06a05316b56371bcdfb588321f9d197080eb29d5